sklearn/metrics/pairwise.py

```diff
@@ -164,44 +197,32 @@ def euclidean_distances(X, Y=None, Y_norm_squared=None, squared=False,
     """
     Considering the rows of X (and Y=X) as vectors, compute the
     distance matrix between each pair of vectors.
-
     For efficiency reasons, the euclidean distance between a pair of row
     vector x and y is computed as::
-
```

Comment:
Well if you removed it it doesn't exist. It existed before, but not after your PR. This is not an artifact of github.